### PR TITLE
Enable gzip interceptors dynamically

### LINKED ISSUE #29730: Provide option to dynamically enable/disable CXF GZIP interceptors for JAX-WS clients
Some Liberty customers need an option to dynamically enable/disable GZIPOutInterceptor and GZIPInInterceptor for a Web Services JAX-WS client. They need an option to enable/disable GZIPOutInterceptor on outbound SOAP request and also should be able to enable/disable GZIPInInterceptor on inbound response message.

### PR DESCRIPTION
**Pull request by sawadood**
- [ X] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [X ] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [X ] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Fixes #29730

### CODE REVIEW

**Comment by sawadood** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.jaxws.2.2.webcontainer_fat, com.ibm.ws.jaxws.2.2.webcontainer_fat_extended, com.ibm.ws.jaxws.cdi_fat, com.ibm.ws.jaxws.clientcontainer_fat, com.ibm.ws.jaxws.ejb_fat, io.openliberty.jaxrs.2.1.jaxws.2.2_fat, io.openliberty.jaxws.beanvalidation_fat,  io.openliberty.jaxws.global.handler.internal_fat, io.openliberty.jaxws.security_fat, io.openliberty.jaxws.security_fat.1, io.openliberty.jaxws.security_fat.ssl, io.openliberty.jaxws.tools_fat, io.openliberty.jaxws.wsat_fat, io.openliberty.jaxws_fat, com.ibm.ws.wsat.common_fat, com.ibm.ws.wsat.concurrent_fat, com.ibm.ws.wsat.migration_fat.1, com.ibm.ws.wsat.migration_fat.2, com.ibm.ws.wsat.migration_fat.3, com.ibm.ws.wsat.migration_fat.4, com.ibm.ws.wsat.migration_fat.5, com.ibm.ws.wsat.migration_fat.6, com.ibm.ws.wsat.migration_fat.7, com.ibm.ws.wsat.recovery_fat.lps, com.ibm.ws.wsat.recovery_fat.multi.1, com.ibm.ws.wsat.recovery_fat.multi.2, com.ibm.ws.wsat.recovery_fat.multi.3, com.ibm.ws.wsat.recovery_fat.multi.4, com.ibm.ws.wsat.recovery_fat.multi.5, com.ibm.ws.wsat.recovery_fat.multi.6, com.ibm.ws.wsat.recovery_fat.multi.7, com.ibm.ws.wsat.recovery_fat.single.1, com.ibm.ws.wsat.recovery_fat.single.2, com.ibm.ws.wsat_fat.assertion, com.ibm.ws.wsat_fat.db, com.ibm.ws.wsat_fat.db_optional, com.ibm.ws.wsat_fat.dbs, com.ibm.ws.wsat_fat.dbs_optional, com.ibm.ws.wsat_fat.multi, com.ibm.ws.wsat_fat.multi.misroute, com.ibm.ws.wsat_fat.ssl, com.ibm.ws.wsat_fat.ssl.misroute, io.openliberty.jaxws.wsat_fat, com.ibm.ws.wssecurity_fat.wsscxf.1, com.ibm.ws.wssecurity_fat.wsscxf.2, com.ibm.ws.wssecurity_fat.wsscxf.3, com.ibm.ws.wssecurity_fat.wsscxf.4, com.ibm.ws.wssecurity_fat.wsscxf.saml.1, com.ibm.ws.wssecurity_fat.wsscxf.saml.2, com.ibm.ws.wssecurity_fat.wsscxf.saml.3, com.ibm.ws.wssecurity_fat.wsscxf.saml.5, com.ibm.ws.wssecurity_fat.wsscxf.saml.6, io.openliberty.xmlws.4.0.internal_fat

**Review by BerksanAtes — APPROVED** [review]
Looks good to me!

File: dev/io.openliberty.jaxws.config_fat/publish/files/GzipInterceptorsTestServer/server.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<server>
+    <featureManager>
+        <feature>jsp-2.2</feature>
+        <feature>jaxws-2.2</feature>
```
Comment (neuwerk) [inline dev/io.openliberty.jaxws.config_fat/publish/files/GzipInterceptorsTestServer/server.xml:4]: Looking at the build failures, I wonder if this feature list is causing the break. Should verify that the JSP-2.2 feature is the right one to pull in for this test. 

```
Runtime features were not of the expected version for repeat action (Server: EnableDefaultValidationTestServer, Action: EE8_FEATURES).
Expected: servlet-4.0, Actual: servlet-3.1.
```

**Review by neuwerk — CHANGES_REQUESTED** [review]

File: dev/io.openliberty.jaxws.config_fat/publish/files/GzipInterceptorsTestServer/server.xml (hunk 1 above)
Comment (neuwerk) [reply dev/io.openliberty.jaxws.config_fat/publish/files/GzipInterceptorsTestServer/server.xml:4]: Changes are unrelated, you can ignore this.

**Review by neuwerk — APPROVED** [review]

**Pull request merged**